frontend/packages/patternfly/src/components/notification-drawer/notification-drawer.tsx

```diff
@@ -17,8 +17,9 @@ const NotificationDrawer: React.FC<NotificationDrawerProps> = ({
     <Drawer isExpanded={isExpanded} isInline={isInline}>
       <DrawerContent>{children}</DrawerContent>
       <DrawerPanelContent noPadding>
-        <NotificationDrawerHeading>{notificationEntries}</NotificationDrawerHeading>
+        {isExpanded && <NotificationDrawerHeading>{notificationEntries}</NotificationDrawerHeading>}
```

Comment:
Don't we want to skip rendering the entire drawer if it's not open? Not just the sections?

Reply:
@spadgett we want the notification drawer to be active to allow for collecting notifications for the badge icon for both alerts and cluster updates. So we need the cluster version to load for that reason. However, I don't want it to block the app from loading...do you have a suggestion for a workaround for that situation?

Reply:
That's fine, but it looks like you're rendering everything in the drawer except the heading here.

Reply:
@jcaianirh Sure, but don't you want to hide the entire `DrawerPanelContent` not just the heading?

I think removing the `AsyncComponent` should make performance a lot better.

Reply:
I guess hiding `DrawerPanelContent` breaks the layout :(

I think the real issue is the `AsyncComponent`, which slows down page load quite a bit